### PR TITLE
Oauth

### PR DESCRIPTION
**Pull request by tutok**
Documentation update according to changes introduced via [#424](https://github.com/deity-io/falcon/pull/424) PR

### CODE REVIEW

File: docs/backend/installing-magento2.md [hunk 1]
```diff
@@ -49,14 +49,37 @@ You can do so by changing your [Falcon-Server config](miscellaneous/config.md).
...
+        }
       }
     }
   }
 }
 ```
 
+##### Connect Falcon to Magento through oAuth
+
+To connect through oAuth you need to create an new Magento integration:
```
Comment (cane-lagerwaard) [inline docs/backend/installing-magento2.md]:
```suggestion
To connect through oAuth you need to create a new Magento integration:
```

File: docs/backend/installing-magento2.md [hunk 2]
```diff
@@ -49,14 +49,37 @@ You can do so by changing your [Falcon-Server config](miscellaneous/config.md).
...
   }
 }
 ```
 
+##### Connect Falcon to Magento through oAuth
+
+To connect through oAuth you need to create a new Magento integration:
+* sign in to Magento Admin panel
+* go to *System* / *Extensions* / *Integrations* and press *Add new Integration*)
+* create new *Integration* via providing its *Name* and specifying *Resource Access: All*
```
Comment (Borales) [inline docs/backend/installing-magento2.md]: @tutok is it safe to provide `ALL` access? (@sergy-gzh?)

File: docs/backend/installing-magento2.md [hunk 3]
```diff
@@ -49,14 +49,37 @@ You can do so by changing your [Falcon-Server config](miscellaneous/config.md).
...
+* *Activate* an integration, grab all *Integration Tokens* and update Falcon `api-magento2.config.auth` section:
```
Comment (Borales) [inline docs/backend/installing-magento2.md]:
```suggestion
* *Activate* an integration, grab all *Integration Tokens* and update Falcon-Server `apis.magento2.config.auth` config section:
```

File: docs/backend/installing-magento2.md (hunk 2 above)
Comment (tutok) [reply docs/backend/installing-magento2.md]: @Borales what do you think about this:
```suggestion
* create new *Integration* via providing its *Name* and specifying *Resource Access* of your choice.
```

Comment (sergy-gzh) [inline docs/backend/installing-magento2.md]:
```suggestion
* Fill in the *Name* and go to  *Role Resources* tab, set *Resource Access* to *Custom* and check the following permissions in the list below:
** Sales
** Catalog
** Stores -> Settings -> All stores
** Global Search
```

File: docs/backend/installing-magento2.md [hunk 4]
```diff
@@ -49,14 +49,41 @@ You can do so by changing your [Falcon-Server config](miscellaneous/config.md).
...
 }
 ```
 
+##### Connect Falcon to Magento through oAuth
+
+To connect through oAuth you need to create a new Magento integration:
+* sign in to Magento Admin panel
+* go to *System* / *Extensions* / *Integrations* and press *Add new Integration*)
+* Fill in the *Name* and go to  *Role Resources* tab, set *Resource Access* to *Custom* and check the following permissions in the list below:
+** Sales
```
Comment (sergy-gzh) [inline docs/backend/installing-magento2.md]:
```suggestion
    * Sales
```

File: docs/backend/installing-magento2.md [hunk 5]
```diff
@@ -49,14 +49,41 @@ You can do so by changing your [Falcon-Server config](miscellaneous/config.md).
...
+** Catalog
```
Comment (sergy-gzh) [inline docs/backend/installing-magento2.md]:
```suggestion
    * Catalog
```

File: docs/backend/installing-magento2.md [hunk 6]
```diff
@@ -49,14 +49,41 @@ You can do so by changing your [Falcon-Server config](miscellaneous/config.md).
...
+** Stores -> Settings -> All stores
```
Comment (sergy-gzh) [inline docs/backend/installing-magento2.md]:
```suggestion
    * Stores -> Settings -> All stores
```

File: docs/backend/installing-magento2.md [hunk 7]
```diff
@@ -49,14 +49,41 @@ You can do so by changing your [Falcon-Server config](miscellaneous/config.md).
...
+** Global Search
```
Comment (sergy-gzh) [inline docs/backend/installing-magento2.md]:
```suggestion
    * Global Search
```

**Comment by tutok** [comment]
so to be clear, from my perspective his PR can be merged, if you @cane-lagerwaard think the same can you take care of that?

**Pull request merged**